client/src/store/slices/mentor/index.ts

```diff
@@ -57,6 +57,25 @@ export const saveMentor = createAsyncThunk(
   }
 );
 
+export const saveThumbnail = createAsyncThunk(
+  "mentor/saveThumbnail",
+  async (headers: {
+    accessToken: string;
+    mentorId: string;
+    file: File;
+  }): Promise<string | unknown> => {
+    try {
+      return await api
+        .uploadThumbnail(headers.mentorId, headers.file)
```

Comment:
this is just silly to save and then fetch here. Can `uploadThumbnail` return the thumbnail url?